### ISSUE TITLE
electric meter digit not recognized

### ISSUE THREAD

**Issue by toebelemount** [jomjol/AI-on-the-edge-device#211]
I try to read out my Siemens electric meter ("Stromzähler") from the 80s, the software works well and the digits are all recognized but the "6" (digit 4), the software recognizes always a "0". The pre-value with the "6" in it is ignored.
If the "6" appears at digit 6 or 7, it is recognized as "6". Even the reflexions at digit 6 or 7 don't trouble.
Screenshot:
![stromzaehler](https://user-images.githubusercontent.com/83663695/117150217-3cc40880-adb8-11eb-91e2-879d40dd45c6.jpg)
The number set looks like this:
![ziffernset](https://user-images.githubusercontent.com/83663695/117150267-464d7080-adb8-11eb-9bd4-80ba0c6c0d52.jpg)
If you are not too busy, there is a ZIP of my logged digits, 2 or 3 each. Maybe that would solve my problem if you integrate it!
The wrong "6" is marked with "WRONG" in the filename.
[digits.zip](https://github.com/jomjol/AI-on-the-edge-device/files/6427481/digits.zip)

**Comment by jomjol** [comment]
Will be used in the next traiing session over the weekend

**Comment by toebelemount** [comment]
Great work! Now all numbers are read perfectly. Thanks so much!
There is no space for the ESP in front of my counter (cabinet door!), so I installed a mirror (coated plexiglass) and let the camera look from above. The software supports "mirror" and "180 degree". WLAN reception was very poor from the inside of the metal cabinet, so I connected an extra antenna. No 3D-print, only some old ABS parts sawn and screwed together.
Because of bad reflections I don't use the built-in LED. I lifted the collector of the transistor near the LED and connected two external LEDs lighting from the left and right side of the counter, flexible with enamelled copper wire ("Kupferlackdraht"), each with a 10 ohm resistor from the collector to the LED-cathode, the LED-anode to +3V3.
Here a sidelong picture:
![esp-electricmeter](https://user-images.githubusercontent.com/83663695/117588974-3d95cb00-b127-11eb-933c-7bb5b0537a2a.jpg)
